packages/storybook-html/src/components/core/colour-base.story.mdx

```diff
@@ -150,7 +150,7 @@ Neutral tones can be used as a background colour.
     </tr>
     <tr className="uqds-table__row">
       <td className="uqds-table__item"><div className="uqds-colour__palette uqds-colour--darkgrey"></div></td>
-      <td className="uqds-table__item" data-title="Name:">UQ darkgrey</td>
+      <td className="uqds-table__item" data-title="Name:">UQ Darkgrey</td>
```

Comment:
Also revert this casing but in this case, add a space between dark and grey.

Reply:
Done